libs/barista-components/stacked-series-chart/src/stacked-series-chart.ts

```diff
@@ -541,7 +541,7 @@ export class DtStackedSeriesChart implements OnDestroy, OnInit {
     legend: DtStackedSeriesChartLegend,
   ): DtStackedSeriesLegendHoverData {
     return {
-      seriesName: legend.label,
+      stackName: legend.label,
```

Comment:
Good catch @tomheller  - yes we need to leave seriesName in there since its part of the public event.
@jsblanco you can add additional fields, but we cannot rename or remove fields on a public API without a breaking change.